easybuild/easyconfigs/c/CGAL/CGAL-4.8.1-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+name = 'CGAL'
+version = '4.8.1'
+gmpver = '6.0.0a'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+# note: source URL needs to be updated for a new version (checksums too)!
```

Comment:
@valtandor is this comment still correct? what part of the source URL would need to be updated?